pkg/chartutil/capabilities.go

```diff
@@ -69,3 +69,55 @@ func (v VersionSet) Has(apiVersion string) bool {
 	_, ok := v[apiVersion]
 	return ok
 }
+
+// Default versions as of 1.14, excluding deprecated versions, focused on workloads
+func defaultVersions() []string {
```

Comment:
Is there a lib in kubernetes that lists all of the default API groups? That way we wouldn't have this large list of default API versions laying around.

Reply:
I'll poke around and see

Reply:
Looks like there is an API for that and it provides 554 items to stick in the default set.